go.work

```diff
@@ -1,3 +1,7 @@
 go 1.20
```

Comment:
note: I omitted (and will continue to omit) this file from source control but I'm open to some tooling to help out folks w pointing go.work files at the main tree.